kodilla-rps/src/main/java/com/kodilla/rps/statistics/Statistics.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.rps.statistics;
+
+public class Statistics {
+
+    private int userRoundResult;
```

Comment:
po co te dwa pierwsze pola? Ja bym poprostu dal:
`private int roundNumber; // numer aktualnej rundy (wiemy więc, ile rund już bylo rozegranych)`
`private int wins; // ile razy Player wygrał`
`private int losts // ile razy player przegrał, a więc komputer wygrał`
Uwaga: jeśli w enumie masz WIN, LOSE - to tutaj zrób analogicznie wins, losts; Jeśli w enumie zrobisz PLAYER, COMPUTER - to tutaj plaerWins, computerWins :) Bądźmy konsekwentni